whipper/test/test_common_config.py

```diff
@@ -66,3 +66,28 @@ def testDefeatsCache(self):
         defeats = self._config.getDefeatsCache(
             'PLEXTOR ', 'DVDR   PX-L890SA', '1.05')
         self.assertEquals(defeats, True)
+
+    def test_get_musicbrainz_server(self):
+        # Test default return value
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          'musicbrainz.org')
+
+        # Section setup
+        self._config._parser.add_section('musicbrainz')
+
+        # Test normal URL
+        self._config._parser.set('musicbrainz', 'server',
+                                 '192.168.2.141:5000')
+        self._config.write()
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          '192.168.2.141:5000')
+
+        # Test URL with added path
+        self._config._parser.set('musicbrainz', 'server',
+                                 '192.168.2.141:5000/hello/world')
+        self._config.write()
+        self.assertEquals(self._config.get_musicbrainz_server(),
+                          '192.168.2.141:5000')
```

Comment:
It might be worth stating that the expected behaviour is for get_musicbrainz_server() to strip the path after the port number in a msg= param, but I don't feel strongly about it.